src/main/java/domain/Car.java

```diff
@@ -0,0 +1,45 @@
+package domain;
+
+public class Car {
+    public String name;
+    public int location;
+
+    public Car(String name) {
+        validator(name);
+        this.name = removeBlank(name);
+        location = 0;
+    }
+
+    private void validator(String name) {
+        validateSize(name);
+        validateBlankName(name);
+    }
+
+    private void validateSize(String name) {
+        if (!(name.length() <= 5)) {
```

Comment:
```suggestion
        if (name.length() > 5) {
```
조건문 앞에 !를 붙이는건 정말 어쩔 수 없을때 붙이는게 좋을것 같아요 혼동을 줄 수 있어서요!